test/testdata/packager/lsp_update_packaged_file_static_field/a.1.rbupdate

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+# typed: strict
+# assert-fast-path: a.rb,__package.rb,dep/exported.rb,dep/__package.rb
+
+class Package::A
+  extend T::Sig
+
+  sig {returns(String)}
+  def self.a
+    Dep::Exports::ExportedClass::Val
```

Comment:
Without this change, the reference on line 10 here reports a resolved but not visible error on the fast path.